c_project/lib/tavildarLDPC/LdpcC/LdpcCode.h

```diff
@@ -286,6 +288,72 @@ class LdpcCode {
             }
         }
 
+        // printf("Codeword:\n");
+        // for (unsigned i = 0; i < _N; i++){
+        //     printf("%i,", codeword.at(i));
+        // }
+
+        return codeword;
+    };
+
+
+    // Encoding defintion for 5G matrices
+    std::vector<uint8_t> encode_modern(std::vector<uint8_t> info_bits) {
```

Comment:
It's unclear how this encode function is different. Ideally the name would be descriptive enough, but a clearer comment would help too